static/js/confirm_dialog.js

```diff
@@ -53,8 +52,11 @@ export function show_confirm_dialog_spinner() {
 }
 
 export function launch(conf) {
-    const html = render_confirm_dialog({fade: conf.fade});
-    const confirm_dialog = $(html);
+    const confirm_dialog_html = render_confirm_dialog({
+        heading_text: conf.html_heading,
```

Comment:
This should be called `heading_html`, if it's being interpolated into the DOM directly as HTML.

(Proper naming for this helps avoid security bugs).